pkg/sql/rowenc/testutils.go

```diff
@@ -998,6 +998,14 @@ func RandEncDatumRowsOfTypes(rng *rand.Rand, numRows int, types []*types.T) EncD
 //  - string (converts to DString)
 func TestingMakePrimaryIndexKey(
 	desc catalog.TableDescriptor, vals ...interface{},
+) (roachpb.Key, error) {
+	return TestingMakePrimaryIndexKeyForTenant(desc, keys.SystemSQLCodec, vals...)
+}
+
+// TestingMakePrimaryIndexKeyForTenant is the same as TestingMakePrimaryIndexKey, but
+// allows specification of the codec to use when encoding keys..
```

Comment:
nit: extra `.` at the end. Probably not worth a second round of CI, but wanted to point out in case other changes are going to be made.